tests/unit/src/test/scala/tests/DiagnosticsLspSuite.scala

```diff
@@ -329,4 +329,42 @@ class DiagnosticsLspSuite extends BaseLspSuite("diagnostics") {
       )
     } yield ()
   }
+
+  test("tokenization-error") {
```

Comment:
This test also passes without these changes, do you think there is a way to assert we don't get stale errors like 
```stale bloop error: unclosed string literal val n: Int = "``` ?

Reply:
That doesn't seem easy to do, so I'd rather not delve into that since at the worst it's just a log message